src/ethereum/ethMiningFees.js

```diff
@@ -32,6 +32,22 @@ export function calcMiningFee(
     const { customNetworkFee } = spendInfo || {}
     if (spendInfo.networkFeeOption === ES_FEE_CUSTOM && customNetworkFee) {
       const { gasLimit, gasPrice } = customNetworkFee
+      const {
+        gasLimit: { minGasLimit },
+        gasPrice: { minGasPrice }
+      } = currencyInfo.defaultSettings.otherSettings.defaultNetworkFees.default
+      if (
+        gasLimit < minGasLimit ||
+        gasPrice < minGasPrice ||
+        isNaN(gasLimit) ||
+        isNaN(gasPrice)
+      ) {
+        // TODO: Throw proper InvalidCustomFee error
+        throw new Error(
+          `Gas Limit: ${minGasLimit} Gas Price (Gwei): ${minGasPrice}`
```

Comment:
This does not look like a clean error. Instead create an error object and set `e.name` to `ErrorBelowMinimumFee`